src/typings/dcos__http-service.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Observable } from "rxjs";
+
+interface RequestOptions {
```

Comment:
IMHO we should add these to the package, its ours, so we can do it quick 🤔

Reply:
Yes, we should. For me this is out of scope for this ticket.

Reply:
Think we should them create a ticket to move that for the package.